nav2_bt_navigator/README.md

```diff
@@ -66,6 +67,10 @@ returns FAILURE, all nodes are halted and this node returns FAILURE.
 
 * SpeedController: A custom control flow node, which controls the tick rate based on the current speed.  This decorator offers the most flexibility as the user can set the minimum/maximum tick rate which is adjusted according to the current speed.
 
+* GoalUpdater: A custom control node, which updates the goal pose. It subscribes to a topic in which it can receive an updated goal pose to use instead of the one commanded in action.
+
+* TruncatePath: A custom control node, which modifies a path making it shorter. It removes the last part of the pose in the base of a distance to the original goal pose. The resulting last pose of the path orientates the robot to the original goal pose.
```

Comment:
```suggestion
* TruncatePath: A custom control node, which modifies a path making it shorter. It removes parts of the path closer than a distance to the goal pose. The resulting last pose of the path orientates the robot to the original goal pose.
```

Reply:
Is that accurate? Is the last pose in the path still the goal not the goal - distance? I don't think that's right

Reply:
yes!! better ;) Thanks